README.md

```diff
@@ -8,6 +8,121 @@
   yarn add @artsy/react-responsive-media
 ```
 
+## Basic Usage
+
+```js
+import { createMedia } from "@artsy/fresnel"
+
+const ExampleAppMedia = createMedia({
+  breakpoints: {
+    xs: 0,
+    sm: 768,
+    md: 1000,
+    lg: 1200,
+  },
+})
+
+const { Media, MediaContextProvider } = ExampleAppMedia
+
+const App = () => {
+  return (
+    <MediaContextProvider>
+      <Media at="xs">Hello mobile!</Media>
+      <Media greaterThan="xs">Hello desktop!</Media>
+    </MediaContextProvider>
+  )
+}
+```
+
+## Server-side Rendering (SSR) Usage
+
+First, configure `@artsy/fresnel` in a `Media` file that can be shared across the app:
+
+```js
```

Comment:
Probably doesn’t make an actual difference, but seeing as we use TS and the example filename is `tsx`, perhaps change these to `tsx`?